project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.4.4
```

Comment:
You may want to add the `.bsp` folder to `.gitignore` or it will start adding it as a change to commit every time you build the code locally.